azure-functions/acbs-function/__mocks__/durable-functions.js

```diff
@@ -1,9 +1,12 @@
+const actualDurableFunctions = jest.requireActual('durable-functions');
+
 const mockClient = {
   startNew: jest.fn(() => 'mockInstanceId'),
   createCheckStatusResponse: jest.fn(),
 };
 
 module.exports = {
+  ...actualDurableFunctions,
```

Comment:
Test was failing before because `df.input.durableClient()` was called in acbs-http/index - I dont think anything else is called on `df` in the existing tests

Reply:
I could just add a mock for that but I don't _think_ it's necessary